python/nano/test/openvino/basic/test_openvino.py

```diff
@@ -26,3 +26,11 @@ def test_openvino_model(self):
         x = np.random.randn(1, 3, 224, 224)
         y_hat = openvino_model.forward_step(x)
         assert tuple(next(iter(y_hat)).shape) == (1, 1000)
+
+    def test_openvino_model_async_predict(self):
+        openvino_model = OpenVINOModel("./intel/resnet18-xnor-binary-onnx-0001/FP16-INT1/resnet18-xnor-binary-onnx-0001.xml")
+        x = [np.random.randn(1, 3, 224, 224) for i in range(5)]
```

Comment:
I think a more common case is that `x =np.random.randn(5, 3, 224, 224) ` ?

Reply:
It's not a batch of data, line #32 means there are 5 inputs and we are going to do inference 5 times. So I think it's better to use a list of arrays than concatenate them together?